altair/utils/tests/test_data.py

```diff
@@ -63,3 +66,39 @@ def test_type_error():
     for f in (sample, limit_rows, to_values):
         with pytest.raises(TypeError):
             pipe(0, f)
+
+
+def test_to_json():
```

Comment:
Maybe also a simple test that covers the dict/values path?

Reply:
Yep, good idea.